tests/time_of_flight/unwrap_test.py

```diff
@@ -178,7 +182,10 @@ def test_unwrap_with_no_choppers() -> None:
         distance=distance,
         choppers=choppers,
         simulation=time_of_flight.simulate_beamline(
-            choppers=choppers, neutrons=300_000, seed=1234
+            choppers=choppers,
+            source_position=sc.vector([0, 0, 0], unit='m'),
```

Comment:
```suggestion
            source_position=fakes.source_position(),
```

Reply:
I didn't do this because `fakes.source_position` corresponds to `fakes.*_choppers` and we don't use those functions here. But I guess the source position actually does not matter here.